queue-api/src/main/java/com/example/queueapi/application/service/QueueService.java

```diff
@@ -54,6 +57,40 @@ private String createToken(String queue, Long userId) {
 
     public long getRankUser(String queue, String token) {
         Long rank = stringRedisTemplate.opsForZSet().rank(USER_QUEUE_WAIT_KEY.formatted(queue), token);
+        if (rank == null) {
+            return -1;
```

Comment:
```suggestion
            long DEFAULT_RANK = -1;
            return DEFAULT_RANK;
```

`-1` 이라는 의미가 명확하지 않아 보여요 ㅎㅎ

의미를 주는 변수명을 갖는건 어떨까요?